### ISSUE TITLE
Thank you - query about porting.

### ISSUE THREAD

**Issue by clort81** [thedeemon/dirtcleaner#1]
Thank you for this important work.

I can not find a similar filter for linux (ffmpeg).  This amazes me, considering how important this function is for treating old film material.

I will attempt to draw developer attention to dirtcleaner and help us get similar functionality under linux.  Do we have permission to use your source?

**Comment by thedeemon** [comment]
Yes, please feel free to port. Sorry I didn't provide a license file. I'll add the MIT one.
